test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -573,7 +573,8 @@ Describe 'help renders when using a PAGER with a space in the path' -Tags 'CI' {
         Set-Content -Path $fakePagerPath -Value $fakePager
 
         $SavedEnvPager = $env:PAGER
-        $env:PAGER = $fakePagerPath
+        $fakePagerCustomArgs = "here is a fake argument"
+        $env:PAGER = "`"$fakePagerPath`" `"$fakePagerCustomArgs`""
```

Comment:
```suggestion
        $env:PAGER = '"$fakePagerPath" "$fakePagerCustomArgs"'
```

Reply:
That won't interpolate variables. Try instead:
```suggestion
        $env:PAGER = '"{0}" "{1}"' -f $fakePagerPath, $fakePagerCustomArgs
```